src/components/ExperienceEditor/ExperienceEditor.tsx

```diff
@@ -222,6 +222,12 @@ export const ExperienceEditor: React.FC<ExperienceEditorProps> = props => {
     }
   };
 
+  const renderLabelButton = () => {
+    if (type === 'Edit') return 'Save changes';
+    else if (type === 'Clone') return 'Clone Experience';
+    else return 'Create Experience';
```

Comment:
Removing `else` would produce the same results. Could you check this? Either way it's okay.